evap/results/templates/results_course_detail_questionnaires.html

```diff
@@ -12,7 +12,14 @@ <h4>{{ section.questionnaire.public_name }}</h4>
 <table class="table table-striped{% if not last %} mb-3{% endif %}">
     <tbody>
         {% for result in section.results %}
-            {% if result.question.is_rating_question %}
+            {% if result.question.is_heading_question %}
+                {# We want to start a new section here, so we close the table, render the heading and reopen the table #}
+                </tbody>
+                </table>
+                <h5{% if not forloop.first %} class="mt-3"{% endif %}>{{ result.question.text }}</h5>
+                <table class="table table-striped{% if not last %} mb-3{% endif %}">
+                <tbody>
+            {% elif result.question.is_rating_question %}
```

Comment:
@karyon: Do you like the indentation this way? I feel like there is no way to not make it look ugly or misleading...

Reply:
Besides that, all requested changes were done, so if you approve, we can merge this.

Reply:
yeah that's probably as good as it will get.